.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -5,17 +5,21 @@ title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
-**What kind of business use case are you trying to solve? What are your requirements?**
-A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
 
-**What is the problem? What is preventing you from meeting the requirements?**
-Describe what impact this problem has for lack of a solution.
+### What are you proposing?
+_In a few sentences, describe the feature and its core capabilities._
```

Comment:
I see most features are born from problems. 
I would always love to see the problem and how the feature would solve it in a proposal.

Reply:
Would it make sense to replace _How did you come up with this proposal?_ with _What problem are you trying to solve?_

Reply:
@elfisher consider @saratvemulapalli's comment?

Reply:
I like the framing of "What problems are you trying to solve?"